### PR TITLE
v0.30.0 Release

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: _posts/2023-07-12-v0.30-local-dependencies-and-enhanced-externals.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+
+One way I will be using this feature is to created a package of shared types and
+functions that I can use in both the Gleam Erlang backend and the Gleam
+JavaScript frontend of my web applications.
+
+
+## Enhanced externals
+
+Gleam can without any overhead import and call functions written in other
+languages on the runtime such as Erlang, Elixir, Erlang, and TypeScript.
```
Comment (aslilac) [inline _posts/2023-07-12-v0.30-local-dependencies-and-enhanced-externals.md:44]: Erlang twice

File: _posts/2023-07-12-v0.30-local-dependencies-and-enhanced-externals.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    "erlang" "halt"
+}
+
+if javascript {
+  external fn native_halt(Int) -> Nil =
+    "./my_javascript_module.mjs" "halt"
+}
+```
+
+A public function with a documenation comment calls a private function, and
```
Comment (aslilac) [inline _posts/2023-07-12-v0.30-local-dependencies-and-enhanced-externals.md:66]: documentation

Comment (lpil) [reply _posts/2023-07-12-v0.30-local-dependencies-and-enhanced-externals.md]: You ninja!

File: book/javascript/highlightjs-gleam.js [hunk 3]
```diff
@@ -74,10 +73,22 @@ hljs.registerLanguage("gleam", function (hljs) {
           },
         ],
       },
+      {
+        className: "attribute",
```
Comment (inoas) [inline book/javascript/highlightjs-gleam.js:77]: This needs to go into the general highlight js and into the compiler for the docs gen, right? Could we submodule this?

File: book/javascript/highlightjs-gleam.js [hunk 4]
```diff
@@ -74,10 +73,22 @@ hljs.registerLanguage("gleam", function (hljs) {
...
+        begin: "@",
+        end: "\\(",
+        excludeEnd: true,
+      },
       {
         className: "keyword",
         beginKeywords: KEYWORDS,
       },
+      {
+        // Attributes
```
Comment (inoas) [inline book/javascript/highlightjs-gleam.js:87]: same

File: javascript/highlightjs-gleam.js [hunk 5]
```diff
@@ -74,10 +73,22 @@ hljs.registerLanguage("gleam", function (hljs) {
           },
         ],
       },
+      {
```
Comment (inoas) [inline javascript/highlightjs-gleam.js:76]: another copy of the same thing?

File: javascript/highlightjs-gleam.js [hunk 6]
```diff
@@ -74,10 +73,22 @@ hljs.registerLanguage("gleam", function (hljs) {
...
+        begin: "@",
+        end: "\\(",
+        excludeEnd: true,
+      },
       {
         className: "keyword",
         beginKeywords: KEYWORDS,
       },
+      {
+        // Attributes
```
Comment (inoas) [inline javascript/highlightjs-gleam.js:87]: see straight above

File: book/javascript/highlightjs-gleam.js (hunk 3 above)
Comment (inoas) [reply book/javascript/highlightjs-gleam.js:77]: https://github.com/gleam-lang/gleam-highlight.js/pull/4

Comment (inoas) [reply book/javascript/highlightjs-gleam.js:77]: https://github.com/gleam-lang/gleam/pull/2267

**Comment by inoas** [comment]
Do we need changes for the treesitter plugin for the new attribute syntax?

**Pull request merged**